eslint-config/rules/plugin-jsdoc.js

```diff
@@ -0,0 +1,247 @@
+module.exports = {
+  plugins: [
+    'jsdoc',
+  ],
+  rules: {
+    /**
+     * Checks that `@access` tags are valid.
+     */
+    'jsdoc/check-access': ['off'],
+
+
+    /**
+     * Reports invalid alignment of JSDoc block asterisks.
+     */
+    'jsdoc/check-alignment': ['warn'],
+
+
+    /**
+     * Ensures that (JavaScript) examples within JSDoc adhere to ESLint rules.
+     */
+    'jsdoc/check-examples': ['warn'],
+
+
+    /**
+     * Reports invalid padding inside JSDoc blocks.
+     */
+    'jsdoc/check-indentation': ['warn'],
+
+
+    /**
+     * Ensures that parameter names in JSDoc match those in the function declaration.
+     */
+    'jsdoc/check-param-names': ['warn'],
+
+
+    /**
+     * Ensures that property names in JSDoc are not duplicated on the same block and that nested properties have defined roots.
+     */
+    'jsdoc/check-property-names': ['off'],
+
+
+    /**
+     * Reports against Google Closure Compiler syntax.
+     */
+    'jsdoc/check-syntax': ['off'],
+
+
+    /**
+     * Reports invalid block tag names.
+     */
+    'jsdoc/check-tag-names': ['warn', {
+      definedTags: ['endpoint'],
+    }],
+
+
+    /**
+     * Reports invalid value types.
+     */
+    'jsdoc/check-types': ['warn'],
+
+
+    /**
+     * Checks the values of specific metadata tags.
+     *
+     * https://github.com/gajus/eslint-plugin-jsdoc/blob/master/README.md#check-values
+     */
+    'jsdoc/check-values': ['off'],
+
+
+    /**
+     * Expects specific tags to be empty of any content.
+     *
+     * https://github.com/gajus/eslint-plugin-jsdoc/blob/master/README.md#empty-tags
+     */
+    'jsdoc/empty-tags': ['off'],
+
+
+    /**
+     * Reports an issue with any non-constructor function using @implements.
+     */
+    'jsdoc/implements-on-classes': ['off'],
+
+
+    /**
+     * Enforces a regular expression pattern on descriptions.
+     */
+    'jsdoc/match-description': ['off'],
+
+
+    /**
+     * Enforces a consistent padding of the block description.
+     */
+    'jsdoc/newline-after-description': ['off'],
+
+
+    /**
+     * Checks for multi-line-style comments which fail to meet the criteria of a jsdoc block.
+     */
+    'jsdoc/no-bad-blocks': ['off'],
```

Comment:
I feel like this rule should be enabled to ensure compatibility for all parsers.

Reply:
No objections to this one, my IDE itself will error this so I did not think to enable it myself

Reply:
Done!